Microsoft.Toolkit.Uwp.Services/Microsoft.Toolkit.Uwp.Services.csproj

```diff
@@ -1,5 +1,12 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
+  <PropertyGroup Label="Globals">
```

Comment:
Why is this change needed?

Reply:
I'm pretty sure VS added that automatically at some point...i will go back and remove.